### PR TITLE
tenable_io: update integration based on feedback from Tenable

### LINKED ISSUE #7671: [Tenable.io] Tweaks to Tenable.io integration
Based on feedback from Tenable, some tweaks need to be applied in order to get our integration validated. 

- **Asset & Vuln data streams pull frequency** : Adjust the default interval to 24 hours and adjust titles from 'logs' to 'data', e.g. Collect asset logs from Tenable.io becomes ' Collect asset data from Tenable.io'

- **Plugin Data Pull** - Adjust initial & daily pull to 24hr, set to default off

- **Scanner Pull** - Remove

- **Scans pull** - Adjust cadence to 24hr, set to default off

Tenable has also rebranded Tenable.io to Tenable Vulnerability Management.  Can please adjust the title, description and all references of Tenable.io to Tenable Vulnerability Management. 

The logo for the integration also needs to be changed:
Tenable-Logo2021

### PR DESCRIPTION
**Pull request by efd6**
## What does this PR do?

* adjust collection intervals to 24 hourly
* make plugin and scans data stream disabled by default
* remove scanner data stream
* adjust branding

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

- Close #7671

## Screenshots

Screenshot 2023-09-07 at 17 47 55

Screenshot 2023-09-07 at 17 52 13

### CODE REVIEW

File: packages/tenable_io/data_stream/asset/manifest.yml [hunk 1]
```diff
@@ -1,16 +1,16 @@
-title: Collect Asset logs from Tenable.io
+title: Collect Asset data from Tenable.io
```
Comment (bhapas) [inline packages/tenable_io/data_stream/asset/manifest.yml:1]: Should we not change title to `Tenable Vulnerability Management`

File: packages/tenable_io/manifest.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 format_version: 2.7.0
 name: tenable_io
 title: Tenable.io
```
Comment (bhapas) [inline packages/tenable_io/manifest.yml:3]: The title here? Should we rebrand?

**Review by bhapas — APPROVED** [review]
LGTM

Can you add some screenshots of the integrations UI page with the rebranding of this integration

**Comment by bhapas** [comment]
@efd6 The first screenshot still shows lot of instances of `tenable.io` references? Should we not change them in documentation?

**Comment by efd6** [comment]
Browser cache. PTAL. There are remaining instances in the dashboards.

**Comment by MakoWish** [comment]
Why was the `scanner` dataset removed? I added that one.

**Comment by efd6** [comment]
This was removed because of feedback from Tenable. It is in the work list in #7671. @jamiehynds Can you comment on this?

**Comment by MakoWish** [comment]
Yeah, it just says "remove", but no real reasoning behind it.

**Pull request merged**